src/effects/How to make an effect stateless.txt

```diff
@@ -35,7 +39,7 @@ Let use use EffectEcho as an example.
     add class definition in the .cpp with mSettings:
     
         struct EffectEcho::Validator
-           : DefaultEffectUIValidator
+           : EffectUIValidator
```

Comment:
The initialization of the base class below also changes the class name

Reply:
oops that's right